CHANGELOG.md

```diff
@@ -1,3 +1,79 @@
+### citus v9.0.0 (October 7, 2019) ###
+
+* Adds support for PostgreSQL 12
+
+* Adds UDFs to help with PostgreSQL upgrades
+
+* Delegates `SELECT function()` to a worker node
+
+* Distributes types to worker nodes
+
+* Introduces `create_distributed_function` UDF
+
+* Introduces local query execution for Citus MX
+
+* Implements infrastructure for routing `CALL` to MX workers
+
+* Adds a feature flag to turn off `CREATE TYPE` propagation
+
+* Adds option `citus.single_shard_commit_protocol`
+
+* Adds schemas when distributing sequence definitons correctly
+
+* Adds support for `EXPLAIN SUMMARY`
+
+* Adds support for `GENERATE ALWAYS AS STORED`
+
+* Adds support for `serial` and `smallserial` when syncing metadata
+
+* Adds support for anon composite types on the target list in router queries
+
+* Avoids a failed assertion when using an aggregate named `invalid`
+
+* Avoids race condition between `create_reference_table` & `master_add_node`
+
+* Avoids undefined behavior of formatting null with `%s`
+
+* Disallows distributed functions for functions depending on an extension
+
+* Fixes a bug in `pg_depend_recurse`
+
+* Fixes a bug that caused `run_command_on_colocated_placements` to fail
+
+* Fixes a bug that leads to various issues when a connection is lost
+
+* Fixes a schema leak on `CREATE INDEX` statement
+
+* Fixes an error message when cluster fails for task tracker
```

Comment:
This is too uncommon case to have in the release notes. I think we can remove this.